### PR TITLE
Add filter by languages to data server requests

### PR DESCRIPTION
**Pull request by smacker**
Ref: https://github.com/src-d/lookout-sdk/issues/36

Signed-off-by: Maxim Sukharev <max@smacker.ru>

### CODE REVIEW

**Comment by smacker** [comment]
I'm ok with both names. @carlosms wdyt?

**Comment by carlosms** [comment]
I agree with @dpordomingo, makes sense to keep consistency and opens the possibility to have `exclude_languages` if we later need it.

**Comment by smacker** [comment]
👍 renamed

File: proto/lookout/sdk/service_data.proto [hunk 1]
```diff
@@ -75,4 +76,5 @@ message FilesRequest {
     bool want_uast = 6 [(gogoproto.customname) = "WantUAST"];
     // WantLanguage set to true if UAST was requested
     bool want_language = 7;
+    repeated string include_languages = 9;
```
Comment (carlosms) [inline proto/lookout/sdk/service_data.proto]: why 9 instead of 8?

Comment (smacker) [reply proto/lookout/sdk/service_data.proto]: copy-past from ChangesReq.
Fixed. Thanks!

**Pull request merged**